### DISCUSSION TITLE
Infinite missions upon reloading

### DISCUSSION THREAD

**Discussion by RooneyRoll** [FakeFishGames/Barotrauma#13879] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When all the possible missions are taken from an outpost if I save and then load the game the mission giving npc have new set of missions which I am able to accept.

### Reproduction steps

1. Dock and arrive at a station or outpost
2.  Go to the top and get the missions you can choose from
3. Save and quit the game
4. Load the game and repeat from step 2.

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Linux

**Comment by Regalis11** [comment]
Thank you for the report! I [created a ticket about this](https://github.com/FakeFishGames/Barotrauma/issues/13880) in the issue tracker, we'll look into it!

**Comment by Jasontti** [comment]
Tested the fix and no issues found.
Closing as completed.
